Sources/JWT/Signers/RSASigner.swift

```diff
@@ -26,67 +26,117 @@ extension HashMethod {
     }
 }
 
-public struct RS256: RSASigner {
-    public let key: Bytes
+public typealias CRSAKey = UnsafeMutablePointer<RSA>
```

Comment:
Totally agree. I think the `key` property originally had to be public because of the `Key` protocol (thus forcing all of this stuff to be public as well). Now that the `Key` protocol is gone, I should be able to make all of this private again.